Cargo.toml

```diff
@@ -29,6 +29,7 @@ oid4vp = { git = "https://git@github.com/impierce/openid4vc.git", rev = "12fed14
 async-trait = "0.1"
 axum = { version = "0.7", features = ["tracing"] }
 base64 = "0.22"
+chrono = "0.4"
```

Comment:
We actually need the `serde` feature for `chrono` (I am actually surprised why the compiler does not complain about it :thinking:) :
```rust
chrono = { version = "0.4", features = ["serde"] }
```

Reply:
We need it for )(de)serializing:
```rust
#[derive(Debug, Serialize, Deserialize, Clone)]
#[serde(untagged)]
pub enum CredentialExpiry {
    Fixed(chrono::DateTime<chrono::Utc>),
    Relative(iso8601_duration::Duration),
    Never,
}
```